packages/assets/scss/01-atoms/_figure.scss

```diff
@@ -77,15 +77,15 @@
     &.align-left {
       float: left;
       margin-left: 0;
-      width: 50%;
+      width: 47%;
```

Comment:
why are you adjusting the width? This is going to create some visual regressions https://42639-137521197-gh.circle-artifacts.com/0/backstop/reports/html/index.html

Reply:
When there are two medium sized images floated together, it doesn't fit in one row since both of them are 50% of width plus some margin. But I see what you mean, I guess I can add this adjustment on OpenMass just for image paragraphs so nothing else is affected. I'll move it there.

Reply:
I reverted this line and added an override  to OpenMass instead to address specifically this case, see: https://github.com/massgov/openmass/pull/1059